### PR TITLE
Added trigger_http variable to repository-function submodule

### PR DESCRIPTION
**Pull request by Nathan-Gauci-Kasna**
Similar to PR: https://github.com/terraform-google-modules/terraform-google-event-function/pull/56
Docs have been regenerated for this change.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@Nathan-Gauci-Kasna
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by Nathan-Gauci-Kasna** [comment]
@morgante please let me know if there are any issues with this PR or you need further info.

File: modules/repository-function/main.tf [hunk 1]
```diff
@@ -21,12 +21,21 @@ resource "google_cloudfunctions_function" "main" {
...
-    resource   = var.event_trigger["resource"]
+  dynamic "event_trigger" {
+    for_each = var.trigger_http != null ? [] : [1]
+
+    content {
+      event_type = var.event_trigger["event_type"]
+      resource   = var.event_trigger["resource"]
+
+      failure_policy {
+        retry = var.event_trigger_failure_policy_retry
```
Comment (bharathkkb) [inline modules/repository-function/main.tf:36]: This var is not defined

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR and sorry for the delay.

**Comment by comment-bot-dev** [comment]
@Nathan-Gauci-Kasna
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/repository-function/main.tf (hunk 1 above)
Comment (Nathan-Gauci-Kasna) [reply modules/repository-function/main.tf:36]: Thanks for the feedback, I will correct this and let you know

**Comment by comment-bot-dev** [comment]
@Nathan-Gauci-Kasna
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/repository-function/main.tf (hunk 1 above)
Comment (Nathan-Gauci-Kasna) [reply modules/repository-function/main.tf:36]: @bharathkkb This has been corrected, please let me know if there are any other issues

**Comment by comment-bot-dev** [comment]
@Nathan-Gauci-Kasna
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**